src/components/dropdown/Dropdown.tsx

```diff
@@ -136,7 +136,11 @@ type AsyncProps<T> = {
   options?: undefined;
   /** Load the options that populate the dropdown from a returned promise */
   asyncOptions: (inputValue: string) =>Promise<DropdownOption<T>[]>;
-  defaultOptions?: boolean;
+  /**
+   * The default set of options to show before the user starts searching. When
+   * set to `true`, the results for asyncOptions('') will be autoloaded.
+   */
+  defaultOptions?:  DropdownOption<T>[] | boolean;
```

Comment:
What happens if we provide a list of options? and what happens if we provide a boolean?

Reply:
- If we provide a list of options, they will be loaded when the dropdown is rendered. 
- If we provide a boolean: 
   - > 'true' will load asyncOptions when the dropdown is rendered
   - > 'false' will do nothing